apps/socket/src/handlers/tic-tac-toe.handler.ts

```
@@ -0,0 +1,291 @@
+		const player = this.socket.data.user as SocketUser;
+		if (!player) {
+			return this.failure("Unauthorized user");
+		}
+
+		const room = ticTacToeStore.getRoom(roomId);
+		if (!room) {
+			return this.failure("Room not found");
+		}
+
+		if (room.isFull) {
+			return this.failure("Room is full");
```

Biome (1.9.4)

[error] The function should not return a value because its return type is void.

The function is here:

'void' signals the absence of value. The returned value is likely to be ignored by the caller.

(lint/correctness/noVoidTypeReturn)